tailwind.config.js

```diff
@@ -2,7 +2,12 @@
 module.exports = {
   content: ["./*.{html,js}"],
   theme: {
-    extend: {},
+    extend: {
+      colors:{
+        ourPrimaryGreen: "#00672E",
+        bgPrimary: "EFECEA"
```

Comment:
You may have forgotten to add # in front of the hexadecimal color.

Reply:
Thank you, I have added the #